projects/ngqp/core/src/test/standalone-textarea.spec.ts

```diff
@@ -0,0 +1,118 @@
+import { Component } from '@angular/core';
```

Comment:
I think this test file doesn't add anything relevant, it just uses a different component. It certainly makes sense to test the different component types for which we have ControlValueAccessor implementations, but not in the scope of standalone params. I think we can just remove this file (unless I'm missing something?)

Reply:
Deleted.